nullifiersCounter/nullifiersCounter.circom

```diff
@@ -20,6 +21,11 @@ template BuildNullifier() {
     blinder ==> hasher.inputs[1];
     salt ==> hasher.inputs[2];
 
+    // verify document commitment value
+    component commitmentHasher = Poseidon(1);
+    commitmentHasher.inputs[0] <== blinder;
+    commitmentHasher.out === commitment;
```

Comment:
Why add public input and then add "commitmentHasher.out === commitment;" constraint when you can set output do commitment <== commitmentHasher.out. Thus, you should not calculate the hash value to pass it to the circuit